src/components/transactions/Switch/SwitchTxSuccessView.tsx

```diff
@@ -41,8 +88,44 @@ export const SwitchTxSuccessView = ({
           textAlign: 'center',
         }}
       >
-        <Typography>
-          <Trans>You&apos;ve successfully switched tokens.</Trans>
+        <Typography mb={2}>
+          {provider === 'cowprotocol' ? (
+            <>
+              {orderStatus === 'open' ? (
+                <Trans>
+                  You&apos;ve successfully submitted an order.
+                  <br /> Please wait for it to be filled.
+                </Trans>
+              ) : orderStatus === 'failed' ? (
+                <Trans>The order has been cancelled.</Trans>
+              ) : (
+                <Trans>The order has been filled.</Trans>
+              )}
+              <br />
+              {txHash && (
+                <>
+                  You can see the details{' '}
```

Comment:
I feel we should tell the user here maybe they can also be seen in tx history instead of going to cow maybe...